src/gui/DatabaseWidget.cpp

```diff
@@ -1520,13 +1524,30 @@ void DatabaseWidget::onGroupChanged()
 
 void DatabaseWidget::onDatabaseModified()
```

Comment:
It is a general question related to UI standard. The best practice is to mark the modified document by `*` (star) marker. Because the saving is delayed now the end user will be confused not seeing modified marker on the current database. What about to change the title of the current DB by `*`?

Reply:
This might indicate you have an error in your implementation. You should treat a delay in saving as having no auto save at all (ie, the modified signal `*` shows immediately in the title). If that is not happening then you are blocking critical signals that trigger that to happen.

Reply:
I checked, seems it works correctly:
![Changed db is marked](https://user-images.githubusercontent.com/121412908/218803164-6b0691f3-de12-4708-a0da-21fb6472e113.png)

Reply:
Checked on linux, i do get the `*` to show when the save is delayed, did you use another os maybe when the `*` didn't show?

Reply:
I checked on Linux too. I don't think your changes will impact on this functionality on other OSs.